### DISCUSSION TITLE
The "HEALTH" of the "cephcluster" is always HEALTH_WARN?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10344] [Q&A]
Exec: ceph -s
![image](https://user-images.githubusercontent.com/13997135/170676965-dba7dcad-d35c-4473-a839-534a22169f4c.png)

Exec: kubectl get cephcluster -n rook-ceph
![image](https://user-images.githubusercontent.com/13997135/170677108-e8c78d06-d3e6-476a-acc1-e41b17c43c6d.png)

The lasteChecked is: 2022-05-25, but today is: 2022-05-27
![image](https://user-images.githubusercontent.com/13997135/170677297-c804303f-e03d-494d-b0fa-79cf98fafda2.png)

@travisn Why the HEALTH of cephcluster is not updated?

**Comment by travisn** [comment]
This health status is expected to be updated every 60s by the operator. Is the operator running, or any errors in the operator log? If the main reconcile of the cluster fails, the health check will not update. But it looks like your cluster is healthy. Also curious if restarting the operator will cause it to update as well, perhaps there is a bug with the health checker stopping unexpectedly.

**Reply by zhucan** [reply]
I will take a deep look at the code. @travisn

**Comment by zhucan** [comment]
![image](https://user-images.githubusercontent.com/13997135/170912965-12885f04-a1e2-4635-b31c-3a72064bedb0.png)
maybe we need to use the patch to update the status.
